client/rollup.config.js

```diff
@@ -42,7 +42,8 @@ export default [
         include: ['src/utils/config.js'],
         preventAssignment: false,
         __api_url__: '/api', // set in firebase.json
-        __purchase_mode__: process.env.AVOCANO_PURCHASE_MODE,
+        __purchase_mode__: process.env.AVOCANO_PURCHASE_MODE || 'default',
```

Comment:
non-blocking: Not sure if "default" is used elsewhere, but if we are surfacing this value in the footer maybe we should use a more descriptive name, such as `buynow`